assembly/assembly-che-tomcat8/src/assembly/bin/setenv.sh

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
license checker may report issue on the date ?

Reply:
yes. https://github.com/eclipse/che/issues/14531